deploy/vagrant/provision/roles/ufs_gcs/files/config_alluxio.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+sed -i "s|^export ALLUXIO_UNDERFS_ADDRESS=.*|export ALLUXIO_UNDERFS_ADDRESS=gs://${GCS_BUCKET}|g" /alluxio/conf/alluxio-env.sh
```

Comment:
in general, this is a very bad mechanism for configuring alluxio because it relies on the verbatim text of the alluxio-env.sh script .... the correct way to do this is to make sure that hte alluxio-env.sh script is invoked using an environment that already has the ALLUXIO_UNDERFS_ADDRESS variable set to the GCS_BUCKET

in other words, it is okay to assume that the environment variables used by the alluxio-env.sh script will not change, but it is not okay to assume that the script itself will never change